plugins/dev-tools/src/addGUIControls.js

```diff
@@ -292,6 +300,9 @@ function saveGUIState(guiState, defaultToolboxName) {
   if (guiState.toolboxName !== defaultToolboxName) {
     hashGuiState.toolbox = guiState.toolboxName;
   }
+  if (guiState.themeName !== defaultThemeName) {
```

Comment:
Why do you have to check this? Why can't you always set it, even if it is the default value?

Reply:
If the value is set, we add url parameters (eg: theme=classic), and open any relevant GUI controls folders.

Same reasoning for not including all options unless selected by the user, eg: rtl=false, toolboxPosition=start, etc.

Reply:
Seems reasonable. I was just curious, thanks!